browser_history/generic.py

```diff
@@ -337,15 +356,41 @@ def to_csv(self):
                 writer.writerow(row)
             return output.getvalue()
 
-    def to_json(self, json_lines=False):
+    def to_json(self, json_lines:bool = False) -> str:
         """
         Return history or bookmarks formatted as a JSON or JSON Lines format
-        names
+        names. If ``json_lines`` flag is `True` convert to JSON Lines format,
+         otherwise convert it to Plain JSON format.
```

Comment:
Indentation error that sphinx complains of
```suggestion
        otherwise convert it to Plain JSON format.
```